packages/compiler/src/core/logger/console-sink.browser.ts

```diff
@@ -10,9 +10,26 @@ export function createConsoleSink(options: any): LogSink {
 
   return {
     log,
+    trackAction: (message, finalMessage, action) => trackAction(message, finalMessage, action),
   };
 }
 
 export function formatLog(log: ProcessedLog): string {
   return JSON.stringify(log);
 }
+
+async function trackAction<T>(
+  message: string,
+  finalMessage: string,
+  asyncAction: () => Promise<T>,
+): Promise<T> {
+  // eslint-disable-next-line no-console
+  console.log(message);
+
+  try {
+    return await asyncAction();
+  } finally {
+    // eslint-disable-next-line no-console
+    console.log(`✓ ${finalMessage}`);
```

Comment:
I think might need to also catch the error and report the x message instead here